src/pydvl/influence/inversion.py

```diff
@@ -31,15 +27,16 @@ class InversionMethod(str, Enum):
 
     Direct = "direct"
     Cg = "cg"
+    Lissa = "lissa"
 
 
 def solve_hvp(
     inversion_method: InversionMethod,
     model: TwiceDifferentiable[TensorType, ModelType],
-    x: TensorType,
-    y: TensorType,
+    training_data: DataLoaderType,
     b: TensorType,
     lam: float = 0,
+    inversion_method_kwargs: Dict[str, Any] = {},
     progress: bool = False,
```

Comment:
I think making `progress` a keyword only argument makes sense. It is completely optional and users would have to explicitly set it.

```suggestion
    *,
    progress: bool = False,
```